windows/scripts/Install-DotNetSdk.ps1

```diff
@@ -1,6 +1,4 @@
-choco install dotnetcore-sdk --version 2.1.302
-choco install dotnetcore-sdk --version 2.1.402
-choco install dotnetcore-sdk --version 2.1.403
-choco install dotnetcore-sdk --version 3.0.100
-choco install dotnetcore-sdk --version 3.1.201
-choco install dotnet-sdk --version 5.0.100
+choco install dotnetcore-sdk --version 2.1.815
+choco install dotnetcore-sdk --version 3.0.103
```

Comment:
do we need `3.0` ? given it's not supported anymore...

Reply:
I guess we can remove it and update the ITs

Reply:
could you FYI the sonar-security folks to be aware ? and maybe open a PR for the update? (I think we should have the roll-forward in global.json files to avoid having to always change the version)

Reply:
Yes, I've talked already with Zuzu about this. And I have https://github.com/SonarSource/sonar-security/pull/2497/files which validates the build with this new image version.

So far it worked well without any changes but if this changes due to the removal of dotnet 3.0 I will take care of the changes.